src/style.css

```
@@ -0,0 +1,116 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  margin: 0 auto 40px;
+  display: block;
+  width: 160px;
+  height: 134px;
+}
+  
+.card__title {
+  margin: 0 0 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+}
+  
+.card__phrase {
+  margin: 0;
+  margin-bottom: 14px;
```

```suggestion
  margin-bottom: 16px;
```
По дизайну тут би мало бути 16 пікселів, а не 14. Можливо, тому й не проходять тести